media/css/sandstone/lib.less

```diff
@@ -52,7 +53,7 @@
 @linkBlueHover:      rgb(0,83,159);
 @linkSkyBlue:        rgb(103,167,208);
 @linkSpaceBlue:      rgb(47,138,202);
-@linkSpacePurple:    rgb(147,111,188);
+@linkSpaceOrange:    rgb(255,173,31);
```

Comment:
I'm not a UX or design person, but I find it a bit hard to distinguish what is and isn't a link here?

<img width="1071" alt="links" src="https://cloud.githubusercontent.com/assets/400117/26194935/f360a92c-3bb1-11e7-9e2f-2532cf62d178.png">

I think we should probably simplify this theme so both the body copy and headings are the same color (white), and then choose a single link color. Rather than add more new colors to our now (already out of date) sandstone pallet, I think we should probably choose from the existing link colors. If we change the page to have a dark themed footer (as seen on the [homepage](https://www.mozilla.org/en-US/), then we can just make the link color blue to match for example.

Thoughts?